src/ConnectedMode.UnitTests/Usings.cs

```diff
@@ -0,0 +1,3 @@
+﻿global using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
I think it's a good idea to have these on a global usings file so we do not need to add on all classes. I did not add the copyright header because we have no "real code" here.